docs/pages/management/dynamic-resources/teleport-operator-helm.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Kubernetes Operator in teleport-cluster Helm chart
+description: Deploy the operator alongside your Helm-deployed Teleport Cluster.
+---
+
+This guide explains how to run the Teleport Kubernetes Operator alongside a Teleport cluster
+deployed via the `teleport-cluster` Helm chart.
+
+If your Teleport cluster is not deployed using the `teleport-cluster` Helm chart
```

Comment:
Can you add this a warning and specify the cloud as an example?

Reply:
I put this in a warning, but the Teleport Cloud example is the next line I think 😅

Reply:
I mean move everything into admonition warning 

```
<Admonition type="warning">

If your Teleport cluster is not deployed using the `teleport-cluster` Helm chart
(Teleport Cloud, manually deployed, deployed via Terraform, ...), you need to follow
[the standalone operator guide](./teleport-operator-standalone.mdx) instead.

</Admonition>
```